main.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'event_info'
+require 'date'
+require_relative 'calendar'
+
+obj = Calendar.new
+loop do
+  begin
+    choice = 0
```

Comment:
why `choice` has been initialized with zero when it will be overwritten always at line number 16?